docs/adr/assets/ADR-003/examples/nodejs/README.md

```diff
@@ -0,0 +1,33 @@
+# Example: Get GitHub App access token in Node.js (TypeScript)
+
+Dependencies are listed in the `package.json` file.
+
+Prepare environment:
+
+```bash
+export GITHUB_APP_ID=...
+export GITHUB_APP_PK_FILE=...
+export GITHUB_ORG="nhs-england-tools"
+```
+
+Run script:
+
+```bash
+$ cd docs/assets/adr/ADR-003/examples/nodejs
+$ npm install
+$ npm start -s
```

Comment:
Do we want to make a decision on whether to use npm or yarn?
Maybe it's worth providing both examples?